openreview/venue/invitation.py

```diff
@@ -1509,6 +1509,231 @@ def set_public_comment_invitation(self):
+                'script': self.invitation_edit_process              
+            }],
+            edit={
+                'signatures': [venue_id],
+                'readers': [venue_id],
+                'writers': [venue_id],
+                'content': {
+                    'noteNumber': {
+                        'value': {
+                            'param': {
+                                'regex': '.*', 'type': 'integer'
+                            }
+                        }
+                    },
+                    'noteId': {
+                        'value': {
+                            'param': {
+                                'regex': '.*', 'type': 'string'
+                            }
+                        }
+                    }
+                },
+                'replacement': True,
+                'invitation': {
+                    'id': self.venue.get_invitation_id('Chat_Reaction', '${2/content/noteNumber/value}'),
+                    'signatures': [ venue_id ],
+                    'readers': ['everyone'],
+                    'writers': [venue_id],
+                    'invitees': comment_stage.get_chat_invitees(self.venue, number='${3/content/noteNumber/value}'),
+                    'cdate': comment_cdate,
+                    'tag': {
+                        'id': {
+                            'param': {
+                                'withInvitation': self.venue.get_invitation_id('Chat_Reaction', '${5/content/noteNumber/value}'),
+                                'optional': True
+                            }
+                        },
+                        'forum': '${3/content/noteId/value}',
+                        'replyto': { 
+                            'param': {
+                                'withForum': '${5/content/noteId/value}', 
+                            }
+                        },                       
+                        'ddate': {
+                            'param': {
+                                'range': [ 0, 999999999999 ],
+                                'optional': True,
+                                'deletable': True
+                            }
+                        },
+                        'signatures': {
+                            'param': { 
+                                'items': [ { 'prefix': s, 'optional': True } if '.*' in s else { 'value': s, 'optional': True } for s in comment_stage.get_chat_signatures(self.venue, '${7/content/noteNumber/value}')]
+                            }                            
+                        },
+                        'readers': comment_stage.get_chat_readers(self.venue, '${4/content/noteNumber/value}'),
+                        'writers': [venue_id, '${2/signatures}'],
+                        'tag': {
+                            'param': {
+                                'enum': ['thumbs-up', 'thumbs-down']
```

Comment:
I think we should directly use the emoji characters in the API. Like:
```suggestion
                                'enum': ['👍', '👎', '😄', '😂', '🔥']
```

Reply:
If we are using UTF-8 character encoding everywhere, which I believe is the case, there should be no problem storing it in the DB or displaying it in the UI.